wails-frontend/src/pages/passwordPrompt.tsx

```diff
@@ -9,24 +9,27 @@ import { events, promptAction, promptRequest } from '../events/events';
 import { useLocation, useNavigate } from 'react-router-dom';
 import { handleApplyResult, handleCancel } from '../utils/utils';
 
+import { FiLock } from 'react-icons/fi';
+import { Password, Button } from '@massalabs/react-ui-kit';
+
 const PasswordPrompt = () => {
   const nav = useNavigate();
 
   const { state } = useLocation();
   const req: promptRequest = state.req;
 
-  const applyStr = (req: promptRequest) => {
+  function applyStr(req: promptRequest) {
```

Comment:
you don't like arrow functions ?
Or is there a decision made to explicitely not use it ?

Reply:
@peterjah We decided in the frontend team to replace const by function.
For example :
- onClick
- onChange
- etc...

But with values like `useState` or simple attribution like `var test = "this is a test";`, we use const.

Reply:
Done